canonical-websites/maas.io
# Remove the old homepage and replace it with the winning variant from the A/B test

## Done

 - Removed the content of index.html with index2.html (the winner)
 - Removed index2.html

## QA

Go to the homepage and see that it now has the content that was previously at index2.html

## Issue / Card

Fixes: https://github.com/CanonicalLtd/MAAS-design/issues/381
